float64 ankle_X_pos
float64 ankle_Y_pos
---
float64 theta1
float64 theta2